doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,361 @@
+| ...           | ...     | ...        | ...       | ...     | ...   |
+
+
+### Sequence Classification for Text Pair UDF
+This UDF takes two input sequences and compares them, e.g., it is used to 
+determine if two sequences are paraphrases of each other. An example usage is given below:
+```sql
+SELECT TE_SEQUENCE_CLASSIFICATION_TEXT_PAIR_UDF(
+    device_id,
+    bucketfs_conn,
+    sub_dir,
+    model_name,
+    first_text,
+    second_text
+)
+```
+- Parameters:
+  - ```device_id```: To run on GPU, specify the valid cuda device ID. Otherwise, 
+  you can leave this field blank.
+  - ```bucketfs_conn```: The BucketFS connection name 
+  - ```sub_dir```: The directory where the model is downloaded in the cache.
+  - ```model_name```: The name of the model to be downloaded. You can find the 
+  details of the models in [huggingface models page](https://huggingface.co/models).
+  - ```first_text```: The first input text
+  - ```second_text```: The second input text
+
+The inference results are presented with predicted _LABEL_ and confidence 
+ _SCORE_ columns, combined with the inputs used when calling this UDF.
+
+
+### Question Answering UDF
+This UDF extracts answer(s) from a given question text. With the `top_k` 
+input parameter, up to `k` answers with the best inference scores can be achieved. 
+An example usage is given below:
+```sql
+SELECT TE_QUESTION_ANSWERING_UDF(
+    device_id,
+    bucketfs_conn,
+    sub_dir,
+    model_name,
+    question,
+    context_text,
+    top_k
+)
+```
+- Parameters:
+  - ```device_id```: To run on GPU, specify the valid cuda device ID. Otherwise, 
+  you can leave this field blank.
+  - ```bucketfs_conn```: The BucketFS connection name 
+  - ```sub_dir```: The directory where the model is downloaded in the cache.
+  - ```model_name```: The name of the model to be downloaded. You can find the 
+  details of the models in [huggingface models page](https://huggingface.co/models).
+  - ```question```: The question text
+  - ```context_text```: The context text, associated with question
+  - ```top_k```: The number of answers to return. Note that, `k` number of answers are not guaranteed. If there are not enough options 
+in the context, it might return less than `top_k` answers (see the [top_k parameter of QuestoinAnswering](https://huggingface.co/docs/transformers/main_classes/pipelines#transformers.QuestionAnsweringPipeline.__call__.topk)).
+
+The inference results are presented with predicted _ANSWER_ and confidence 
+ _SCORE_ columns, combined with the inputs used when calling this UDF.
+If `top_k` > 1, each input row is repeated for each answer. For example:
+
+| BUCKETFS_CONN | SUB_DIR | MODEL_NAME | QUESTION   | CONTEXT   | TOP_K | ANSWER   | SCORE |
+| ------------- | ------- | ---------- |------------|-----------| ----- |----------| ----- |
+| conn_name     | dir/    | model_name | question_1 | context_1 | 2     | answer_1 | 0.75  |
```

Comment:
can you create a ticket, that we add the rank of the answer to the result, this makes selection easier

Reply:
creted https://github.com/exasol/transformers-extension/issues/47